tests/test_data/ingoodflavor.com/ingoodflavor_1.json

```diff
@@ -0,0 +1,55 @@
+{
+  "author": "In Good Flavor",
+  "canonical_url": "https://ingoodflavor.com/2017/10/25/cheesy-linguica-and-potato-bake/",
+  "site_name": "In Good Flavor",
+  "host": "ingoodflavor.com",
+  "language": "en-US",
+  "title": "Cheesy Linguica and Potato Bake",
+  "ingredients": [
+    "1 tbsp. extra virgin olive oil",
+    "2 cups chopped onions",
+    "2 cloves garlic, minced",
+    "2 (about 14 -16 oz each) packages ground Portuguese linguica",
+    "4 (15 oz.) cans sliced potatoes, drained well and coarsely chopped",
+    "3/4 tsp. Lawry’s Seasoned Salt",
+    "1/2 tsp. black pepper",
+    "10 oz. (about 5 cups) shredded sharp cheddar cheese"
+  ],
+  "ingredient_groups": [
+    {
+      "ingredients": [
+        "1 tbsp. extra virgin olive oil",
+        "2 cups chopped onions",
+        "2 cloves garlic, minced",
+        "2 (about 14 -16 oz each) packages ground Portuguese linguica",
+        "4 (15 oz.) cans sliced potatoes, drained well and coarsely chopped",
+        "3/4 tsp. Lawry’s Seasoned Salt",
+        "1/2 tsp. black pepper",
+        "10 oz. (about 5 cups) shredded sharp cheddar cheese"
+      ],
+      "purpose": "Ingredients"
```

Comment:
Nitpick: this `purpose` field seems redundant - I think we'd generally expect/want `null` on a single-group recipe.